website/docs/r/compute_region_instance_group_manager.html.markdown

```diff
@@ -13,6 +13,8 @@ of homogeneous Compute Engine virtual machine instances from a common instance
 template. For more information, see [the official documentation](https://cloud.google.com/compute/docs/instance-groups/distributing-instances-with-regional-instance-groups)
 and [API](https://cloud.google.com/compute/docs/reference/latest/regionInstanceGroupManagers)
 
+~> **Note:** Use [google_compute_instance_group_manager](/docs/providers/google/r/google_compute_instance_group_manager.html) to create a single-zone instance group manager.
```

Comment:
Is '~>' valid markdown? I've never seen that before

Reply:
If it's not I'd prefer to drop it as the ** emphasis should be enough

Reply:
It is an extension of markdown. It creates a block with a light yellow background. You can see an example here:
https://www.terraform.io/docs/providers/google/r/compute_network_peering.html

We uses it already in 9 different pages in our docs.

Reply:
Thanks, it's completely new to me.